.github/workflows/ci-test.yml

```diff
@@ -29,3 +29,8 @@ jobs:
 
       - name: Run linter
         run: npm run lint
+
+      - name: Upload test coverage to Codecov
+        uses: codecov/codecov-action@eaaf4bedf32dbdc6b720b63067d99c4d77d6047d # v3.1.4
```

Comment:
Please use tag @v3 so we don't have to keep updating this

Reply:
done